test/errors/fullUsePath-brief.good

```diff
@@ -1,4 +1,5 @@
-warning in fullUsePath.chpl:1: an implicit module named 'fullUsePath' is being introduced to contain file-scope code
-error in fullUsePath.chpl:9: cannot find module or enum 'ToLookFor' for 'use' statement
-  note in fullUsePath.chpl:1: a declaration of 'ToLookFor' is here
-  note in fullUsePath.chpl:1: however, 'use' statements can only be used with modules or enums (and this 'ToLookFor' is not a module or enum)
+fullUsePath.chpl:1: warning: an implicit module named 'fullUsePath' is being introduced to contain file-scope code
+fullUsePath.chpl:8: In module 'M4':
+fullUsePath.chpl:9: error: cannot find module or enum named 'ToLookFor'
+fullUsePath.chpl:1: note: a declaration of 'ToLookFor' is here
+fullUsePath.chpl:1: note: however, 'use' statements can only be used with modules or enums (and this 'ToLookFor' is not a module or enum)
```

Comment:
also pre-existing, but "and this 'ToLookFor' is neither" feels more succinct / less lecture-y.

Reply:
This was done due to practical limitations (the same reporting mechanism is used for use statements and import statements, but one allows only modules, and other modules and enums, so one would read "neither" and one "not one", or something). It can be made work, though; for that, I'd just need to know what to say instead of "neither" in the import case.

Reply:
In the import case, I felt like the repetition was less annoying / making me feel like the error was trying to make me feel bad, but I'm not sure why, psychologically speaking.  Maybe just the repetition of a single noun doesn't seem as lecture-y as the repetition of a conjunction?  Another alternative would be "is not one".